### ISSUE TITLE
SPIKE: Resubmission

### ISSUE THREAD

**Issue by amilash** [raft-tech/TANF-app#1176]

**Comment by ADPennington** [comment]
Based on FY20 data submissions for states and territories (TBD for tribes):
- Approx. 1/3rd submitted partial data at least once. The remainder either submitted data only once or re-submitted data for all (universe of sample) cases. 
- Partial submissions are more common among sample states and those that are migrating to new systems (note: part of this process usually involves temporarily submitting data from multiple systems until the migration is complete)
- Some that submit partial data also use FTANF/SSPMOE executables, which could be used in the short-term to create complete files (until it is replaced) 
 
Given the above, DIGIT team does not anticipate a big burden on STTs if TDP moves away from partial submissions, and the **_team prefers full (re)submissions to help encourage an accurate and complete accounting of the caseload for each quarterly submission_**.  We also value the ability to see how data has changed overtime, and believe this may also be valuable to STTs to help ensure that any re-submitted data addresses issues identified in prior transmission. That said, before committing to this decision, we would like to learn/discuss the following:
- How many STTs are currently in the middle of, planning for, or considering a system migrations in the next 1-2 years? 
-  If migration effort is underway (nice to have info):
   - What is the anticipated timeline?
   - How does STT envision migration impacting ability to create and transmit quarterly files? 
   - What supports, if available, would be useful during this migration period? 
 - The most common issue we've heard is the need to combine records from multiple systems into one file. How could TDP potentially accommodate while still accepting only complete files (e.g. low-tech tool for combining records from 2 files?, 1-pager on tips for combining and transmitting files from both systems?)
 - Should we prepare a report that compares re-submitted files with most recent version (so STTs can see how records changed)? If so, what kind of information should be included in this report? And when should the report be generated (before or after submit)? 
 
Other notes + technical questions:
- UX team may already have a running list of STTs (and related research findings) who recently or are currently migrating, so where possible we should leverage that. 
- For states that submit sample data, we may learn from fedcap research about how these states prepare files for transmission? This could help us assess potential impact. 
- If we generate a comparison report, need to figure out what is being compared. S3 files will, for now, have a 30 day retention period given the SORN. 
- Also, are we retaining rejected files in S3 (i.e. those with parsing blockers)?

**Comment by ADPennington** [comment]
@reitermb i'd like to pair with you on this to develop of list of things we could potentially learn to prepare for moving away from partial resubmissions
